### DISCUSSION TITLE
Type 'MetamaskConnector' is not assignable to type Connector<any,any>

### DISCUSSION THREAD

**Discussion by DmitriiKholodov** [wevm/viem#801] [Question] [answered]
Hello. I am facing an issue with **wagmi** **createConfig** function.

I'd installed latest versions of packages as far as I understand

```
"viem": "^1.2.2",
"wagmi": "^1.3.2"
```

I create config the same as at official wagmi documentation https://wagmi.sh/examples/connect-wallet

I've researched latest changes and I guess it depends on these changes - https://github.com/wagmi-dev/viem/commit/98fd9172045c344429d5a716f497e422d85c0348, but I don't know how to solve it by myself

Do you have any recommendations?

<img width="1177" alt="image" src="https://github.com/wagmi-dev/viem/assets/28441549/18de53a1-6996-4e52-8162-bf36aab4736b">

**Comment by tmm** [comment, marked as answer]
Fixed by https://github.com/wagmi-dev/references/pull/381

**Reply by DmitriiKholodov** [reply]
Thanks!
